index.html

```diff
@@ -267,64 +267,115 @@ <h1>Live video stream on the talks</h1>
 		  </div>
 		</div>
 
 		<div class="span4 speaker">
 		  <div class="speaker-photo">
-		    <img src="images/speakers/TODO.jpg" />
+		    <img src="images/speakers/gabor_csardi.jpg" />
 		  </div>
 		  <div class="speaker-name">
-		    TODO
+		    Gábor Csárdi
 		  </div>
 		  <div class="speaker-affiliation">
-		    TITLE<br>at COMPANY (COUNTRY)
+		    Software Engineer<br>at RStudio (US)
 		  </div>
 		  <div class="speaker-social">
-		    <a href="https://github.com/TODO"><img src="images/social/gh.png"></a>
-		    <a href="https://twitter.com/TODO"><img src="images/social/tw.png"></a>
-		    <a href="https://www.linkedin.com/in/TODO"><img src="images/social/li.png"></a>
+		    <a href="https://github.com/gaborcsardi"><img src="images/social/gh.png"></a>
+		    <a href="https://twitter.com/gaborcsardi"><img src="images/social/tw.png"></a>
+		    <a href="https://www.linkedin.com/in/gaborcsardi"><img src="images/social/li.png"></a>
+                    <a href="http://www.r-pkg.org/search.html?q=gabor+csardi"><img src="images/social/R.png"></a>
 		  </div>
 		  <div class="speaker-bio">
-		    TODO
+		     Gábor has been writing R tools for more than 10 years, and is the co-author of the igraph R package, and the author of several others. He is the main architect of the Metacran services and r-hub, the first major project of the R Consortium. As of Apr, 2017 he joined Hadley Wickham's team at RStudio. Gábor received his Ph.D. in Computer Science from ELTE, Budapest in 2010.
 		  </div>
 		</div>
-		
+              </div>
+              
+              <div class="row">
+
+		<div class="span4 speaker">
+		  <div class="speaker-photo">
+		    <img src="images/speakers/martin_maechler.png" />
+		  </div>
+		  <div class="speaker-name">
+		    Martin Mächler
+		  </div>
+		  <div class="speaker-affiliation">
+		    Professor of Mathematics<br>at ETH Zurich (CH)
+		  </div>
+		  <div class="speaker-social">
+		    <a href="https://r-forge.r-project.org/users/mmaechler/"><img src="images/social/rforge.png" style="height:20px;"></a>
+		    <a href="https://github.com/mmaechler"><img src="images/social/gh.png"></a>
+		    <a href="https://twitter.com/mmaechler"><img src="images/social/tw.png"></a>
+                    <a href="http://stat.ethz.ch/~maechler/"><img src="images/social/rss.png"></a>
+                    <a href="https://www.r-pkg.org/search.html?q=Martin+Maechler"><img src="images/social/R.png"></a>
+		  </div>
+		  <div class="speaker-bio">
+		    Martin is a Mathematician and Statistician, Professor of ETH Zurich, R Core member, Secretary General of the R Foundation, authored more than 20 R packages (such as Matrix, cluster, robustbase, cobs, VLMC, bitops or copula), Emacs ESS Core Developer since 1997 and Project Leader since 2004, author of several books and over 50 scientific journal articles.
```

Comment:
Mathematician **(Ph.D. ETH Z)** and Statistician, Lecturer and Senior Scientist at Seminar für Statistik, ETH Zurich, ...